yjit/src/options.rs

```diff
@@ -82,6 +82,12 @@ pub struct Options {
 
     /// Enable writing /tmp/perf-{pid}.map for Linux perf
     pub perf_map: Option<PerfMap>,
+
+    // Log compilations
+    pub gen_compilation_log: bool,
+
+    // Print compilation log on exit (when gen_compilation_log is also true)
+    pub print_compilation_log: Option<CompilationLogOutput>,
```

Comment:
Nitpick but it seems like these two variables could be one thing.

CompilationLog could have 4 values:
Disabled
Quiet,
Stdout,
File,

Reply:
I'm happy to do it that way. I structurally mirrored what we're doing with stats, where we have `gen_stats` and `print_stats`. I didn't really love having two separate values anyway.

Reply:
I've consolidated the options. I left the runtime stats one as is. I don't know if we want to take a similar pass on those as well, but it's out of scope for this PR in any event.